stable/docker-registry/values.yaml

```diff
@@ -102,3 +102,5 @@ configData:
       enabled: true
       interval: 10s
       threshold: 3
+
+priorityClassName: ""
```

Comment:
I believe setting to "" it will always insert the value into the chart.
What do you see if you do a test deploy of this?

Reply:
Should be fine as it's wrapped into an `if` statement.